src/modules/blockchainApplication/connection/components/RequestSummary/index.js

```diff
@@ -41,32 +45,37 @@ const RequestSummary = ({ nextStep }) => {
   const [senderAccount, setSenderAccount] = useState(null);
   const { session } = useSession();
   const { moduleCommandSchemas } = useCommandSchema();
+  const metaData = useBlockchainApplicationMeta();
   useDeprecatedAccount(senderAccount);
   useSchemas();
 
+  const sendingChainID = request?.chainId.replace('lisk:', '');
+  const tokenData = useAppsMetaTokens({ config: { params: { chainID: sendingChainID } } });
+
   const approveHandler = () => {
     const moduleCommand = joinModuleAndCommand(transaction);
     const transactionJSON = toTransactionJSON(transaction, moduleCommandSchemas[moduleCommand]);
-    // @todo remove the hardcoded normalized
-    // values according to #4824
+    const { recipientChainID } = request?.request?.params ?? {};
+    const sendingChain = metaData.data.data.find((item) => item.chainID === sendingChainID);
+    sendingChain.chainID = sendingChainID;
+    const recipientChain = metaData.data.data.find((item) => item.chainID === recipientChainID);
+    recipientChain.chainID = recipientChainID;
+    const token = tokenData.data.data.length > 0 ? tokenData.data.data[0] : defaultToken;
```

Comment:
```suggestion
    const token = tokenData?.data?.data?.length > 0 ? tokenData.data.data[0] : defaultToken;
```

I would also suggest we don't do this because what I see happening here is that due to state transition of `tokenData` it would be undefined first and this would result to the UI possibly showing a wrong token symbol `LSK`. 

I would advise that until the tokenMeta and chainMeta is obtained/resolved, no symbol is shown and all actions are disabled.

Reply:
Fair point. I'll apply this.